doc/optimal_power_flow.rst

```diff
@@ -288,7 +289,12 @@ For generators with unit commitment you can also specify ramp limits
 at start-up :math:`rusu_{n,s}` and shut-down :math:`rdsd_{n,s}`
 
 .. math::
-  \left[ -rd_{n,s}*u_{n,s,t} -rdsd_{n,s}(u_{n,s,t-1} - u_{n,s,t})\right] \bar{g}_{n,s} \leq (g_{n,s,t} - g_{n,s,t-1}) \leq \left[ru_{n,s}*u_{n,s,t-1} +   rusu_{n,s} (u_{n,s,t} - u_{n,s,t-1})\right]\bar{g}_{n,s}
+
+  \left[ -rd_{n,s}*u_{n,s,t} -rdsd_{n,s}(u_{n,s,t-1} - u_{n,s,t})\right] \bar{g}_{n,s}
+
+  \leq (g_{n,s,t} - g_{n,s,t-1}) \leq 
+
+  \left[ru_{n,s}*u_{n,s,t-1} +   rusu_{n,s} (u_{n,s,t} - u_{n,s,t-1})\right]\bar{g}_{n,s}
```

Comment:
The ramping constraints render right-aligned in this way. One line is to large isn't it?

I suppose it would be fine if all were centered, or maybe if one would get something like `LaTeX`s multline